common/test/package.scala

```diff
@@ -65,6 +66,7 @@ trait TestSettings {
 trait ConfiguredTestSuite extends ConfiguredServer with ConfiguredBrowser with ExecutionContexts {
   this: ConfiguredTestSuite with org.scalatest.Suite =>
 
+  lazy val webClient = new WebClient(BrowserVersion.CHROME)
```

Comment:
There seems to be no discernible difference in calling the controller directly or using WS.url in terms of execution speed.

Tempted to stick with HtmlUnit as the code will be very slightly more portable to future amp pages (as you just need a URL).